functions.php

```diff
@@ -617,3 +617,52 @@ function ($theme_json) {
         return $theme_json->update_with($new_color_palette);
     }
 );
+
+// Registering into the Settings > Writing setting page
```

Comment:
My preference will be, we should move this piece of code to some other place/file like `planet4-master-theme/src/MasterSite.php`, it will be better for future maintenance.